source/Game/LiveActor/MirrorReflectionModel.cpp

```diff
@@ -0,0 +1,47 @@
+#include "Game/LiveActor/MirrorReflectionModel.h"
+#include "Game/Demo/DemoFunction.h"
+
+void MirrorReflectionModel::init(JMapInfoIter const& pMapInfo) {
+    MR::invalidateClipping(this);
+    DemoFunction::registerDemoSimpleCastAllFunction(this);
+    makeActorAppeared();
+    _94 = 0;
+}
+
+/* I'm not sure what to do about the FUN_80262794 call.
```

Comment:
wrap #ifdef NON_MATCHING around both of these commented out functions, these can be figured out later